charts/gardener/charts/runtime/templates/deployment-apiserver.yaml

```
@@ -19,7 +19,12 @@ spec:
   template:
     metadata:
       annotations:
-        checksum/configmap-gardener-audit-policy-config: {{ include (print $.Template.BasePath "/configmap-apiserver-audit.yaml") . | sha256sum }}
+        {{- if .Values.global.apiserver.audit.policy }}
+        checksum/configmap-gardener-audit-policy-config: {{ include (print $.Template.BasePath "/configmap-apiserver-audit-policy.yaml") . | sha256sum }}
+        {{- end }}
+        {{- if .Values.global.apiserver.audit.policy }}
```

```suggestion
        {{- if .Values.global.apiserver.audit.webhook.config }}
```

The `webhook config` does not depend on the policy file.

---

nice catch